specifyweb/frontend/js_src/lib/components/Atoms/Logos.tsx

```diff
@@ -0,0 +1,98 @@
+        />
+      </g>
+    </svg>
+  ),
+  specify: (
+    <svg
+      aria-hidden
+      className={logoClassName}
+      viewBox="0 0 475.6 142.4"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <path
+        d="M110.3 142.4h32v-33c-6.7 14.2-18.1 25.8-32 33z"
+        fill="currentColor"
+      />
+      <linearGradient
+        gradientUnits="userSpaceOnUse"
+        id="B"
+        x1="-.005"
+        x2="95.345"
+        y1=".055"
+        y2="95.405"
+      >
+        <stop offset="0" stopColor="currentColor" stopOpacity="1" />
+        <stop offset="1" stopColor="currentColor" stopOpacity="0.5" />
+      </linearGradient>
+      <path
+        d="M6.7 78.9c0-39.3 31.9-71.2 71.2-71.2 28.4 0 53 16.7 64.4 40.8V.1H0v142.3h45.6C22.5 130.6 6.7 106.6 6.7 78.9z"
+        fill="currentColor"
+      />
+      <path
+        d="M66.9 115.6c-3.3 1.2-7.1 1.8-11.6 1.8-13.8 0-22.6-6.5-26.5-19.5l5.2-1.8c2.9 10.7 10.1 16.1 21.4 16.1 6.5 0 11.5-1.6 15.2-4.8 3.8-3.3 5.7-8.1 5.7-14.3 0-6-2.2-10.9-6.7-14.8-1.2-1.1-3-2.3-5.4-3.7-2.4-1.3-5.5-2.8-9.3-4.5-7.1-3.2-12-6-14.6-8.3-4.4-3.8-6.5-8.8-6.5-15s2.2-11.2 6.7-15c4.4-3.9 9.7-5.8 16.1-5.8 10.6 0 18 3.9 22.4 11.8L74 40c-1.7-2.8-3.7-4.8-6-6.2C65 32 61.2 31 56.5 31c-5 0-9.1 1.4-12.3 4.2-3.5 2.9-5.3 6.8-5.3 11.6 0 4.3 1.8 8 5.5 11.2 2.1 1.7 6.3 4 12.6 6.8 3.9 1.8 7.3 3.5 10.2 5.2s5.2 3.5 6.9 5.2c4.9 4.8 7.3 10.7 7.3 17.9 0 11.4-4.8 18.9-14.5 22.5h0zm67-5.3c-5.2 5.2-11.6 7.8-19.1 7.8-8.6 0-15.8-3.5-21.4-10.6v34.1h-5.2V65.2h5.2V75c5.4-7.1 12.5-10.6 21.4-10.6 7.6 0 13.9 2.6 19 7.7 5.2 5.2 7.8 11.6 7.8 19.1s-2.6 14-7.7 19.1h0zm-3.8-34.4c-4.3-4.3-9.4-6.4-15.3-6.4s-11.2 2.1-15.3 6.3c-4.3 4.3-6.4 9.4-6.4 15.5s2.1 11.1 6.4 15.3c4.3 4.3 9.4 6.4 15.3 6.4s11.2-2.1 15.3-6.3 6.4-9.4 6.4-15.5-2.1-11-6.4-15.3zm24.1 13.4c0 7.1 1.9 12.6 5.7 16.7 3.9 4.2 9.4 6.3 16.2 6.3 3.5 0 6.7-.6 9.6-1.8 3.6-1.5 6.8-4.6 9.7-9.2l4.7 2.1c-3.3 6.2-8.3 10.4-15.1 12.6-2.6.8-5.5 1.3-8.8 1.3-7.8 0-14.2-2.6-19.2-7.7-5.1-5.1-7.7-11.5-7.7-19.3s2.5-14.2 7.6-19.2c5.1-5.1 11.6-7.7 19.4-7.7s14.1 2.4 19.1 7.2c5.1 4.8 7.7 11 7.7 18.6l-48.9.1h0zm35.5-16.5c-3.9-2.8-8.5-4.3-13.6-4.3-10.7 0-17.8 5.3-21.1 15.9h42.2c-.9-4.8-3.4-8.6-7.5-11.6h0zm52.8 44.6c-7.7 0-14.1-2.6-19.3-7.7s-7.7-11.5-7.7-19.3 2.6-14.1 7.7-19.2 11.5-7.7 19.3-7.7c6.6 0 11.9 1.5 15.9 4.5v6.2l-3.3-2.1-3.5-1.9c-3.2-1.1-6.2-1.6-9.1-1.6-6.5 0-11.7 2-15.7 6-4.1 4.1-6.2 9.4-6.2 15.9s2.1 11.6 6.2 15.7 9.4 6.2 15.7 6.2c6 0 11.3-2 15.9-6v6.4c-4.7 3-10 4.6-15.9 4.6h0zm28.5-1v-52h5.2v52H271zm6.9-65.2c0 2.4-1.9 4.3-4.3 4.3s-4.3-1.9-4.3-4.3 1.9-4.3 4.3-4.3 4.3 1.9 4.3 4.3zM313 28.4l-3.4-.4c-4.7 0-7 2.8-7 8.3v28.1h12.3v5.2h-12.3v46.8h-5.2V69.6h-7v-5.2h7V34.8c0-8 4-12 12.1-12 1.8 0 3 .1 3.5.2 1 0 2.1.3 3.4.9v5.2l-3.4-.7h0zm23.8 112.4H331l13.5-29.2-21.9-47.2h5.5l19.4 41 19.1-41h5.7l-35.5 76.4h0zm77.6-23.2l-4.5-2.9 54.7-83H411v-5.2h63.2l-59.8 91.1h0z"
+        fill="currentColor"
+        stroke="currentColor"
+        strokeMiterlimit="10"
+        strokeWidth="1.5"
+      />
+    </svg>
+  ),
+};
```

Comment:
actually, nevermind, I have a better idea:
what if we need to display the logo somewhere outside of react?
I.e, the back-end is using this image right now

how about instead of creating this file, you try out the following:
- keep the short_logo.svg and logo.svg file
- in those files, replace the hardcoded colors with CSS variables, thus allowing you to customize the color. BUT: provide the hardcoded color as the default value for a color. See https://developer.mozilla.org/en-US/docs/Web/CSS/var. So for example, replace `#123456` with `var(--some-variable-name, #123456)`

Now, if back-end uses the image, the image will show up correctly using default colors
But, if displayed on the front-end, the colors will be customizable

Reply:
okay, now I have an even better idea:
don't need to modify short_logo.svg or logo.svg at all
Instead, apply a CSS filter for the image to hue rotate the colors to be of correct accent color - https://tailwindcss.com/docs/hue-rotate
To calculate the degree to which you need to rotate the color, convert the user's accent color and our original brand color to hsl (create a JS helper function for that conversation - look online how to convert from hex to hsl). Then, the hue values of both values can be compared and converted to degrees

Can provide more details if this is not clear